addons/website/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,139 @@
+---Models in module 'website'---
+obsolete model website.redirect (renamed to website.rewrite)
+new model website.rewrite (renamed from website.redirect)
+# DONE: pre-migration: renamed models
+
+new model website.route
+new model website.track
+new model website.visitor
+# NOTHING TO DO
+
+---Fields in module 'website'---
+website      / ir.ui.view               / track (boolean)               : NEW hasdefault
+website      / mrp.document             / key (char)                    : previously in module mrp
+website      / mrp.document             / website_id (many2one)         : previously in module mrp
+website      / mrp.document             / website_url (char)            : previously in module mrp
+website      / res.partner              / is_published (boolean)        : previously in module website_partner
+website      / res.partner              / visitor_ids (many2many)       : NEW relation: website.visitor
+website      / res.partner              / website_published (boolean)   : previously in module website_partner
+website      / res.partner              / website_url (char)            : previously in module website_partner
+website      / res.users                / is_published (boolean)        : previously in module website_partner
+website      / res.users                / website_published (boolean)   : previously in module website_partner
+website      / res.users                / website_url (char)            : previously in module website_partner
+website      / website                  / default_lang_code (char)      : DEL

```

Comment:
Please don't group all the fields under a generic "Nothing to do". Explain for each "problematic one" why is "Nothing to do". I'm not sure here that this should be nothing to do. How was this used in previous version? And now has is this inferred or done?

Reply:
The `code` is taken from `default_lang_id`, and if there is none, then it's taken from `url_code`